### PR TITLE
Update installation notes

### PR DESCRIPTION
**Pull request by mikelehu**
I have updated package's installation notes at README.md file:

- Pkg.add("IRKGaussLegendre.jl")
- Required Julia 1.5 version or higher

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,9 +31,9 @@ precision computations (for accuracy requirements that exceeds double precision
 This package can be installed using
 
 ```julia
-using Pkg
-Pkg.add("https://github.com/mikelehu/IRKGaussLegendre.jl")
-using IRKGaussLegendre
+julia>using Pkg
+julia>Pkg.add("IRKGaussLegendre.jl")
+julia>using IRKGaussLegendre
```
Comment (ChrisRackauckas) [inline README.md]: usually it's best to keep it simple and copy-pastable. I'd just remove the julia>

Comment (ChrisRackauckas) [inline README.md]:
```suggestion
using Pkg
Pkg.add("IRKGaussLegendre.jl")
using IRKGaussLegendre
```

Comment (mikelehu) [reply README.md]: I agree with your suggested change (removing julia>,...). I have created a pull request to understand the GitHub flow.  Once the pull request is accepted, will the changes be merged into the base branch? 
I would also like to tell you that we have proposed a project to a graduate student to add in the implementation of Gauss Legendre event handling and continuous dense output. I hope you find it interesting

Comment (ChrisRackauckas) [reply README.md]: >we have proposed a project to a graduate student to add in the implementation of Gauss Legendre event handling and continuous dense output. I hope you find it interesting

That sounds great. Have you derived a high order interpolation for GL methods? I don't recall one.

Comment (mikelehu) [reply README.md]: I think so.  Do you think we should include any other functionality in IRKGL16?

Comment (ChrisRackauckas) [reply README.md]: Check compatibility with ForwardDiff for automatic differentiation and I think you've covered everything important. I can definitely help out with AD errors if one find one: that's just pushing generic types through.

Comment (mikelehu) [reply README.md]: We don't understand how to add our own interpolation function in IRGL16.
Is there documentation with examples that can help us?

Comment (ChrisRackauckas) [reply README.md]: You just create a callable struct. Here's a few examples: https://github.com/SciML/SciMLBase.jl/blob/master/src/interpolation.jl

Comment (mikelehu) [reply README.md]: We have created a callable struct for own interpolation function (see IRKGLSolver.jl).
[IRKGL16Solver.jl.txt](https://github.com/SciML/IRKGaussLegendre.jl/files/6086110/IRKGL16Solver.jl.txt)

But when we build the solution, should we specify our interpolation function? How?
In this way?
``
  if (dense==true)

     sol= sol=DiffEqBase.build_solution(prob,alg,tt,uu,  destats=destats,retcode= :Success,
        		                    ks=ks ,dense=true,interp=IRKGL16Interp )
else

    sol=DiffEqBase.build_solution(prob,alg,tt,uu, destats=destats,retcode= :Success)

end

``

Comment (ChrisRackauckas) [reply README.md]: yeah relying on constant folding and doing that is good.

Comment (mikelehu) [reply README.md]: When I  try it,  I get the next error: 
`sol = solve(prob,IRKGL16(),dense=true,saveat=1) `
"MethodError: no method matching interp_summary(::Type{IRKGL16Interp})
Closest candidates are:
  interp_summary(!Matched::Nothing) at /home/joseba/.julia/packages/DiffEqBase/elwdl/src/interpolation.jl:38
  interp_summary(!Matched::DiffEqBase.HermiteInterpolation) at /home/joseba/.julia/packages/DiffEqBase/elwdl/src/interpolation.jl:35
  interp_summary(!Matched::DiffEqBase.LinearInterpolation) at /home/joseba/.julia/packages/DiffEqBase/elwdl/src/interpolation.jl:36
  ..."

[Interpolation test.pdf](https://github.com/SciML/IRKGaussLegendre.jl/files/6095939/Interpolation.test.pdf)

Comment (ChrisRackauckas) [reply README.md]: That's an error from the printing: https://github.com/SciML/SciMLBase.jl/blob/master/src/interpolation.jl#L34

Comment (mikelehu) [reply README.md]: Sorry but we are understanding something wrong.
interp_summary(::IRKGL16Interp) function is defined:

`struct IRKGL16Interp{T} <: DiffEqBase.AbstractDiffEqInterpolation
  t::T
  u::T
  ks::Array{T,1} 
end`

`interp_summary(::IRKGL16Interp)="` IRKGL16 `interpolation"`

`(id::IRKGL16Interp)(tvals,idxs,deriv,p,continuity::Symbol=:left)` = interpolation(tvals,id,idxs,deriv,p,continuity)`

Comment (ChrisRackauckas) [reply README.md]: You have to overload the one in DiffEqBase

**Comment by mikelehu** [comment]
Now, it is right. 
I think we will need more help with interpolation issue
Thanks

**Comment by mikelehu** [comment]
I'm going to start studying how to add Event Handling and Callback Functions in IRKGL16
Can you recommend us any documentation about it?

**Comment by ChrisRackauckas** [comment]
That's very much undocumented as it's internals. But most of the packages just have a callback handler:

https://github.com/SciML/OrdinaryDiffEq.jl/blob/master/src/integrators/integrator_utils.jl#L240-L324

and then all of the callback logic is shared defined in DiffEqBase:

https://github.com/SciML/DiffEqBase.jl/blob/master/src/callbacks.jl

**Pull request merged**